DaemonRenderer/Source/Src/Windowing/Screen.cpp

```diff
@@ -131,4 +151,44 @@ DAEvoid Screen::SetGammaRamp(GammaRamp const& in_gamma_ramp) const noexcept
     glfwSetGammaRamp(m_handle, reinterpret_cast<GLFWgammaramp const*>(&in_gamma_ramp));
 }
 
+#ifdef DAEMON_OS_WINDOWS
+
+DAEchar const* Screen::GetWin32Adapter() const noexcept
+{
+    return glfwGetWin32Adapter(m_handle);
+}
+
+DAEchar const* Screen::GetWin32Monitor() const noexcept
+{
+    return glfwGetWin32Monitor(m_handle);
+}
+
+#endif
+
+#pragma endregion
+
+#pragma region Operators
+
+Screen& Screen::operator=(Screen&& in_move) noexcept
+{
+    m_logger      = in_move.m_logger;
```

Comment:
Same as above: both the logger and the handle can be moved too here since they are pointers, the actual objects will remain untouched

Reply:
A pointer is trivially copyable so moving it is less optimized.